standards/tests/test_standards_rendering.py

```diff
@@ -31,3 +31,15 @@ def test_render_lesson_includes_standard(self):
         # Standards Alignment header only shows if there are standards
         self.assertIn('Standards Alignment', response.content)
         self.assertIn('Test Standard', response.content)
+
+    def test_metadata_for_course(self):
```

Comment:
I don't know if the first two tests I wrote actually belong here since they aren't related to standards, but I didn't know where else to put them and it seemed better to have them than to not. @dmcavoy what do you think?

Reply:
Might make a little more sense for the course and unit one to live here: https://github.com/code-dot-org/curriculumbuilder/blob/master/curricula/tests/test_curricula_rendering.py